test/wanda/catalog/catalog_test.exs

```diff
@@ -1,4 +1,4 @@
-defmodule Wanda.CatalogTest do
+defmodule Wanda.Catalog.CatalogTest do
```

Comment:
I don't think we need to change this name and file placement.
This tests `Wanda.Catalog` which is directly in `wanda` folder